package ru.yandex.praktikum.pageObject;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class OrderStatus {
    WebDriver driver;
    private final By notFound = By.xpath(".//*[@alt='Not found']");

    public OrderStatus(WebDriver driver) {
        this.driver = driver;
    }

    public OrderStatus waitLoadOrderStatusPade() {
        new WebDriverWait(driver, Duration.ofSeconds(10))
                .until(ExpectedConditions.presenceOfElementLocated(notFound));
        return this;
    }

    public void waitLoadOrderStatusPage() {
    }
}
